app/src/main/java/com/kickstarter/ui/activities/SetPasswordActivity.kt

```diff
@@ -34,51 +44,52 @@ class SetPasswordActivity : BaseActivity<SetPasswordViewModel.ViewModel>() {
         }
 
         this.viewModel.outputs.progressBarIsVisible()
-            .compose(bindToLifecycle())
-            .compose(Transformers.observeForUI())
+            .observeOn(AndroidSchedulers.mainThread())
             .subscribe {
                 binding.progressBar.isGone = !it
-            }
+            }.addToDisposable(disposables)
 
         this.viewModel.outputs.setUserEmail()
-            .compose(bindToLifecycle())
-            .compose(Transformers.observeForUI())
+            .observeOn(AndroidSchedulers.mainThread())
             .subscribe {
-                binding.setPasswordHint.text = this.environment().ksString()?.format(getString(R.string.We_will_be_discontinuing_the_ability_to_log_in_via_FB), "email", it)
-            }
+                binding.setPasswordHint.text = getEnvironment()?.ksString()?.format(getString(R.string.We_will_be_discontinuing_the_ability_to_log_in_via_FB), "email", it)
+            }.addToDisposable(disposables)
```

Comment:
i know this was in the last iteration of this as well, but do we have a translated string for "email"?

Reply:
We do! i can update in a subsequent PR. Thanks for catching this.